docs/references/dependency-versions.md

```diff
@@ -9,5 +9,6 @@ Rancher Desktop uses several utilities/subsystems as dependencies under the hood
 
 | Rancher Desktop Version | Dependency Versions |
 | ------------- | ---------------- |
-| v1.6.1 | <Version161 /> |
```

Comment:
You are still importing `@site/docs/dependency-versions/v1.6.1.md` above. That file should be deleted.

Reply:
I have removed the 1.6.1 file and the import statement.